plugins/inputs/postgresql/service.go

```diff
@@ -111,33 +110,14 @@ func (p *Service) Start(telegraf.Accumulator) (err error) {
 	// Specific support to make it work with PgBouncer too
 	// See https://github.com/influxdata/telegraf/issues/3253#issuecomment-357505343
 	if p.IsPgBouncer {
-		d := &stdlib.DriverConfig{
-			ConnConfig: pgx.ConnConfig{
-				PreferSimpleProtocol: true,
-				RuntimeParams: map[string]string{
-					"client_encoding": "UTF8",
-				},
-				CustomConnInfo: func(c *pgx.Conn) (*pgtype.ConnInfo, error) {
```

Comment:
Do we no longer need any driver config? 

The [comment on 111 ](https://github.com/influxdata/telegraf/issues/3253#issuecomment-357505343) points to a comment around this.

Also [this thread](https://github.com/jackc/pgx/issues/649#issuecomment-585547940) seems to suggest we need/needed to register data types?

Reply:
In my opinion, the configuration of the driver is no longer needed inside v4. However, I'm not 100% sure and I don't know enough about PGBounder. For this reason, I will first try to create an integration test and clear the corresponding doubts based on this test.

Reply:
That's great thanks